cluster/olm/ceph/assemble/metadata-common.yaml

```diff
@@ -0,0 +1,241 @@
+spec:
+  customresourcedefinitions:
+    owned:
+      - kind: CephCluster
+        name: cephclusters.ceph.rook.io
+        version: v1
+        displayName: Ceph Cluster
+        description: Represents a Ceph cluster.
+      - kind: CephBlockPool
+        name: cephblockpools.ceph.rook.io
+        version: v1
+        displayName: Ceph Block Pool
+        description: Represents a Ceph Block Pool.
+      - kind: CephFilesystem
+        name: cephfilesystems.ceph.rook.io
+        version: v1
+        displayName: Ceph Filesystem
+        description: Represents a Ceph Filesystem.
+      - kind: CephNFS
+        name: cephnfses.ceph.rook.io
+        version: v1
+        displayName: Ceph NFS
+        description: Represents a Ceph NFS interface.
+      - kind: CephObjectStore
+        name: cephobjectstores.ceph.rook.io
+        version: v1
+        displayName: Ceph Object Store
+        description: Represents a Ceph Object Store.
+      - kind: CephObjectStoreUser
+        name: cephobjectstoreusers.ceph.rook.io
+        version: v1
+        displayName: Ceph Object Store User
+        description: Represents a Ceph Object Store User.
+      - kind: Volume
+        name: volumes.rook.io
+        version: v1alpha2
+        displayName: Ceph Volume
+        description: Represents a Ceph volume.
+  displayName: Rook
+  description: |
+    Rook Storage Orchestration for Kubernetes
+
+    Rook runs as a cloud-native service for optimal integration with applications in need of storage, and handles the heavy-lifting behind the scenes such as provisioning and management.
+    Rook orchestrates battle-tested open-source storage technology Ceph, which has years of production deployments and runs some of the worlds largest clusters.
+    Rook is open source software released under the Apache 2.0 license. Rook has an active developer and user community.
+
+    # Before you start
+    https://rook.io/docs/rook/v1.0/k8s-pre-reqs.html
+
+  keywords: ["rook", "ceph", "storage", "object storage", "open source", "block storage", "shared filesystem"]
+  minKubeVersion: 1.10.0
+  maintainers:
+    - name: Red Hat, Inc.
```

Comment:
For this upstream integration, seems like we would want to mention Rook, and perhaps use the email `info@rook.io` as seen in the main rook [github](https://github.com/rook/rook#contact)